### PR TITLE
Topic 112

### PR DESCRIPTION
**Pull request by dmitryuk**
Fixes https://github.com/DamienHarper/auditor/issues/112
Linked with https://github.com/DamienHarper/auditor/pull/118/files

### CODE REVIEW

**Review by DamienHarper — CHANGES_REQUESTED** [review]
Looks good to me, all you have to do is to fix CS

**Comment by dmitryuk** [comment]
I need a new tag of auditor package to make it works
@DamienHarper, can you make a release?

**Comment by DamienHarper** [comment]
@dmitryuk here you go: [2.1.0](https://github.com/DamienHarper/auditor/releases/tag/2.1.0)

File: .php-cs-fixer.php [hunk 1]
```diff
@@ -41,6 +41,7 @@
         'simplified_if_return' => true,
         'simplified_null_return' => false,
         'static_lambda' => true,
+        'get_class_to_class_keyword' => false,
```
Comment (dmitryuk) [inline .php-cs-fixer.php:44]: Don't replace `get_class($entity)` to `$entity::class`

**Comment by dmitryuk** [comment]
@DamienHarper please make a new review, because some other files changed from last one.
Also see `https://github.com/DamienHarper/auditor-docs/pull/11`

**Review by DamienHarper — APPROVED** [review]

**Pull request merged**